specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/storage.json

```diff
@@ -2462,14 +2515,24 @@
           "type": "string",
           "description": "The identity type.",
           "enum": [
-            "SystemAssigned"
+            "SystemAssigned",
+            "UserAssigned",
+            "SystemAssigned,UserAssigned",
+            "None"
```

Comment:
Update IdentityType enum from single value to multiple value is a breaking change for .net SDK.
Since when single value, SDK will auto assign the only value to the enum when new the Identity class.
When multiple value, need manual assign the enum value in new the identity class. The before new Idenetity class code will not work anymore.

Anyway, for .net SDK, we can release a new major version for the breaking